### PR TITLE
byte.decode

### PR DESCRIPTION
**Pull request by yanganto**
- support ascii, utf-8 encoding

### CODE REVIEW

File: vm/src/obj/objbytes.rs [hunk 1]
```diff
@@ -420,6 +422,117 @@ impl PyBytesRef {
...
+                                decode_content.push(replacing_char.unwrap())
+                            }
+                        }
+                    }
+                }
+            }
+            // utf-8 is default encoding
+            _ => {
+                let mut p: u32 = 0u32;
+                let mut remaining_bytes = 0;
```
Comment (coolreader18) [inline vm/src/obj/objbytes.rs]: utf-8 is the default, but cpython throws a `LookupError` if you give it an invalid encoding.

Comment (yanganto) [reply vm/src/obj/objbytes.rs]: Got it, I will implement this.

Comment (yanganto) [reply vm/src/obj/objbytes.rs]: Done :)

**Comment by yanganto** [comment]
@coolreader18 thanks for  your review.

**Pull request merged**